tools/proofers/proof_validate.js

```diff
@@ -44,4 +44,18 @@ window.addEventListener('DOMContentLoaded', function() {
             }
         });
     });
+
+    const conversionMap = {
+        '‘': '\'',
+        '“': '"',
+        '’': '\'',
+        '”': '"',
+        '—': '-'
```

Comment:
The em-dash needs to be replaced with a double hyphen-minus (U+002d), not a single one.

Reply:
I just spotted another potential issue. The cursor is moved to the end of the page after the substitution.

Reply:
Thanks, I should've addressed both of these. Let me know how your testing goes.